### PR TITLE
Add sort by post creation to api

### PR DESCRIPTION
**Pull request by charmata**
Options are asc (oldest) or desc (newest)

### CODE REVIEW

File: controllers/post.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 const Post = require("../models/Post");
 
 exports.findAll = (req, res) => {
-  let filter = {};
+  let filter,
+    sort = {};
```
Comment (dvadz) [inline controllers/post.js:5]: Are you certain that declaration makes filter = {}

**Review by dvadz — CHANGES_REQUESTED** [review]
Please answer the question regarding line 5.

File: controllers/post.js (hunk 1 above)
Comment (charmata) [reply controllers/post.js:5]: it doesn't, fixed in new commit

**Review by dvadz — APPROVED** [review]
Approved.

**Pull request merged**